src/main/java/ch/njol/skript/expressions/ExprClamp.java

```diff
@@ -0,0 +1,97 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Clamp")
+@Description("Clamps one or more values between two numbers.")
+@Examples({
+	"clamp 5 between 0 and 10 # result = 5",
+	"clamp 5.5 between 0 and 5 # result = 5",
+	"clamp 0.25 between 0 and 0.5 # result = 0.25",
+	"clamp 5 between 7 and 10 # result = 7",
+	"clamp (5, 0, 10, 9, 13) between 7 and 10 # result = (7, 7, 10, 9, 10)",
+	"",
+	"set {_clamped::*} to clamp {_values::*} between 0 and 10"
+})
+@Since("INSERT VERSION")
+public class ExprClamp extends SimpleExpression<Number> {
+
+	static {
+		Skript.registerExpression(ExprClamp.class, Number.class, ExpressionType.COMBINED,
+			"clamp %numbers% between %number% and %number%");
```

Comment:
`infinity value`, but I agree!

Reply:
Good idea, I'll look into this later today

Reply:
That's true. Wouldn't `x clamped below 5` -> `max(x, 5)` and `x clamped above 0` -> `min(x, 0)`. I guess it wouldn't be necessary to add this since there is already a simple way to accomplish the same result. Do you guys think it would still be nice to have it anyway?

Reply:
speaking this expression is mostly utility methods I don't see a reason not to add a second hand shortcut for min/max when it makes senses

Reply:
IF we add the expression, yes, but as ayham has brought up it may or may not be worth it to duplicate functionality that's already available with functions and as such, this expression wouldn't be added at all.